src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormDataServiceImpl.java

```diff
@@ -368,8 +391,14 @@ public FormDataResponse updateFormData(FormDataSchema formDataSchema) throws Jso
         formDataDefinition.setCreatedOn(String.valueOf(document.get(CREATED_ON)));
         formDataDefinition.setUpdatedById(String.valueOf(loggedInUserId));
         formDataDefinition.setUpdatedOn(String.valueOf(Instant.now()));
-        mongoTemplate.getCollection(TP_RUNTIME_FORM_DATA +formId).updateOne(filter,Updates.combine(
-                Updates.set(FORM_DATA,formData),Updates.set(VERSION,version),Updates.set(FORM_META_DATA,formMetaData)));
+        try {
+            mongoTemplate.getCollection(TP_RUNTIME_FORM_DATA + formId).updateOne(filter, Updates.combine(
```

Comment:
unnecessary change, please take care not to add them to commit

Reply:
It is important to catch an exceptions that may arise when updating the officialEmail field, particularly in cases where someone may have inadvertently entered a duplicate value . In these cases we have to handle this situation by catching exception